podcastparser.py

```diff
@@ -696,6 +732,8 @@ def parse_pubdate(text):
     'rss/channel/title': PodcastAttr('title', squash_whitespace),
     'rss/channel/link': PodcastAttrRelativeLink('link'),
     'rss/channel/description': PodcastAttr('description', squash_whitespace_not_nl),
+    'rss/channel/podcast:funding': PodcastAttrFromUrl('funding'),
+    'rss/channel/podcast:locked': PodcastAttrExplicit('locked'),
```

Comment:
As gPodder has "locked" as a feature, and it means slightly different things (prevent files from being deleted locally), maybe something like `import_prohibited` or something might be more descriptive?

Reply:
Changed to `import_prohibited`